docs/feature_advanced_keycodes.md

```diff
@@ -27,6 +27,7 @@ These functions allow you to activate layers in various ways. Note that layers a
 * `MO(layer)` - momentarily activates *layer*. As soon as you let go of the key, the layer is deactivated. 
 * `LM(layer, mod)` - Momentarily activates *layer* (like `MO`), but with modifier(s) *mod* active. Only supports layers 0-15 and the left modifiers.
 * `LT(layer, kc)` - momentarily activates *layer* when held, and sends *kc* when tapped.
+* `OSL(layer)` - momentarily activates *layer* until the next key is pressed.
```

Comment:
Could you include a link to the One Shots section?

Reply:
Just did that for ya.